docker/swarmkit
# Support for Windows named pipes

# Support for Windows named pipes

**- What I did**
Added support for mount type **npipe** same way than it is added for engine on: https://github.com/moby/moby/pull/33852

**- How I did it**
New included new **--npipe** switch.

**- How to test it**
On Linux command like this
```
swarmctl service create --name npipe --npipe //./pipe/docker_engine://./pipe/docker_engine --image alpine
```
will give error
```
Error response from daemon: invalid mount config for type \"volume\": invalid mount path: 'type=bind' mount path must be absolute"
```
because name pipe support exists on Windows parser: https://github.com/moby/moby/blob/master/volume/mounts/windows_parser.go

In Windows it should work but I didn't found at least yet how/if it is possible test SwarmKit without Docker.
